CHANGELOG.md

```diff
@@ -35,7 +35,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ##### Dairon
 
-- Changed: In Early Grapple Room: The Slide Jumps have had their difficulty switched. The first one is now Beginner and the second is Intermediate.
+- Changed: In Early Grapple Room: The Slide Jumps have had their difficulties change. The first one is now Intermediate and the second is Advanced.
```

Comment:
The first one is no longer being changed, and the "d" is missing on the word "change"

Reply:
Right of course.